### PR TITLE
[EDU-5095] feat: add delete an edge storage bucket

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Deleting an Edge Storage bucket
+description: Learn how to delete the buckets in your account using the Azion API and Azion CLI.
+permalink: /documentation/products/store/storage/delete-buckets/
+meta_tags: Edge Storage, Storage, Cloud, S3, Bucket, File Storage, Object Storage, API, Azion CLI, Delete Buckets
+namespace: docs_store_journey_storage_delete_buckets
+menu_namespace: storeMenu
+---
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:9]:
```suggestion
```

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+import Tabs from '~/components/tabs/Tabs'
+
+This guide walks you through updating an Edge Storage bucket using the Azion API and Azion CLI. It includes changing the permissions of a bucket, which can be crucial for managing access to your data.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:13]:
```suggestion
This guide walks you through deleting an Edge Storage bucket using the Azion API and Azion CLI. It includes changing the permissions of a bucket, which can be crucial for managing access to your data. 
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:13]: review introduction

Comment (GabrielAzion) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:13]: deleting

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <Fragment slot="tab.api">API</Fragment>
+
+<Fragment slot="panel.cli">
+
+## Requirements 
+
+- [Azion CLI installed](/en/documentation/products/azion-cli/overview/#installing-azion-cli). 
+- [A personal token configured](/en/documentation/devtools/cli/globals/#token).
+
+To update a bucket:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:28]:
```suggestion
To delete a bucket: 
```

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+namespace: docs_store_journey_storage_delete_buckets
+menu_namespace: storeMenu
+---
+
+
+import Tabs from '~/components/tabs/Tabs'
+
+This guide walks you through updating an Edge Storage bucket using the Azion API and Azion CLI. It includes changing the permissions of a bucket, which can be crucial for managing access to your data. 
+
+## Updating a bucket
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:15]: Deleting

Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:15]:
```suggestion
## Deleting a bucket
```

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx (hunk 3 above)
Comment (GabrielAzion) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:28]: delete

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+To update a bucket: 
+
+```bash 
+azion delete edge-storage bucket --name yourbucketname
+```
+
+</Fragment>
+
+<Fragment slot="panel.api">
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:37]:
```suggestion
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+ 
+Run the following `DELETE` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and `bucket_name` with the name of the bucket you want to delete:
+
+```bash
+curl --location --request DELETE 'https://api.azion.com/v4/storage/buckets/bucket_name' \
+--header 'Accept: application/json' \
+--header 'Authorization: Token [TOKEN VALUE]'
+```
+
+Response:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:47]:
```suggestion
You'll receive a response similar to this: 
```

File: src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+## Deleting a bucket
+
+<Tabs client:visible>
+    <Fragment slot="tab.cli">Azion CLI</Fragment>
+    <Fragment slot="tab.api">API</Fragment>
+
+<Fragment slot="panel.cli">
+
+## Requirements
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/delete-bucket.mdx:22]:
```suggestion
### Requirements 
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**